components/brave_wallet_ui/common/api/getKeyringsByType.ts

```diff
@@ -0,0 +1,52 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
we would call this `getKeyringsByType` if it exported only a single function that had that purpose (and name). Seems like this is now a module which offers a variety of methods to access hardware keyrings - so how about the name `hardware_keyrings.ts`?

Reply:
done